src/data-table.component.ts

```diff
@@ -199,11 +222,24 @@ export class DataTableComponent implements AfterViewInit, OnInit, OnDestroy, OnC
 
   ngAfterViewInit() {
     this.matSortService.sortButtonLabel = this.sortButtonLabel.bind(this);
-    // this.trans.set('SORT_BUTTON_LABEL', 'Change sorting for {{id}}', 'en');
-    // this.trans.set('SORT_BUTTON_LABEL', '{{id}} oszlop sorrendjének megváltoztatása', 'hu');
+    // debug localization
+    // this.trans.set('ICELL_DATA_TABLE.SORT_BUTTON_LABEL', 'Change sorting for {{id}}, {{direction}}', 'en');
+    // this.trans.set('ICELL_DATA_TABLE.SORT_BUTTON_LABEL', '{{id}} oszlop sorrendjének megváltoztatása, {{direction}}', 'hu');
   }
 
   ngOnInit() {
+    // If simple array passed make it sortable datasource
```

Comment:
`datasource` input might change so this section should be refactored into eg. a new property setter for `datasource`

Reply:
updated